nda-transform/count_assoc_files.py

```diff
@@ -97,8 +97,8 @@ def count_files(
     submission_id = args.id
     collection_id = args.collection
 
-    print(f"Submission id: {submission_id}")
-    print(f"Collection id: {collection_id}")
+    print(f"submission id: {submission_id}")
+    print(f"collection id: {collection_id}")
```

Comment:
No that is now what I said. Please emend the help message with upper case. Upper case is more appropriate.